src/Type/Comment/Connection/CommentConnectionDefinition.php

```diff
@@ -35,7 +35,7 @@ class CommentConnectionDefinition {
 	 */
 	public static function connection() {
 
-		if ( null === self::$connection ) {
+		if ( null === self::$connection ) :
```

Comment:
ya, I go back and forth on my usage sometimes. . .I usually prefer brackets, but use colon for longer blocks as it's more explicit where it's ending. . .but ya, we can definitely decide on a standard on when to go to colon vs. brackets. . .like a certain size of code block being wrapped or something